src/snapred/backend/service/CalibrationService.py

```diff
@@ -86,10 +88,11 @@ def __init__(self):
         self.registerPath("hasState", self.hasState)
         self.registerPath("checkDataExists", self.calculatePixelGroupingParameters)
         self.registerPath("assessment", self.assessQuality)
+        self.registerPath("index", self.getCalibrationIndex)
         self.registerPath("normalization", self.normalization)
         self.registerPath("normalizationAssessment", self.normalizationAssessment)
         self.registerPath("saveNormalization", self.saveNormalization)
-        self.registerPath("quality", self.readQuality)
+        self.registerPath("loadAssessment", self.readQuality)
```

Comment:
should the method be given the same name as the string endpoint?

Reply:
Good point. On the other hand, this method is closely related to another method: `assessQuality`. Compare
`self.registerPath("assessment", self.assessQuality)` and 
`self.registerPath("loadAssessment", self.readQuality)`
So, to be consistent, we would probably need to change names in both cases.